serverless-inference/serverless-model-registry.ipynb

```diff
@@ -0,0 +1,483 @@
+{
```

Comment:
<p>Is there a link you can provide listing the supported regions? The region support will expand in the future and this list will become outdated.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3252/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
Availability date for all regions is still being confirmed, could we stay that in the future when generally available this will become available across all regions?